packages/twenty-server/src/modules/company/standard-objects/company.workspace-entity.ts

```diff
@@ -102,6 +96,28 @@ export class CompanyWorkspaceEntity extends BaseWorkspaceEntity {
   @WorkspaceIsNullable()
   annualRecurringRevenue: CurrencyMetadata | null;
 
+  @WorkspaceField({
+    standardId: COMPANY_STANDARD_FIELD_IDS.address_deprecated,
+    type: FieldMetadataType.TEXT,
+    label: 'Address (deprecated) ',
```

Comment:
@Bonapara @FelixMalfait are you ok with this? or would you want a different naming? (it's the label so they can still change it again)
![image](https://github.com/twentyhq/twenty/assets/51697796/60d4985b-b95e-431e-b48b-8806437f3a6a)

Reply:
Address (Deprecated) seems good to me! Ideally, I wonder if we could also prepend the field description with:

"This standard field has been deprecated and migrated as a custom field. Please consider using the new 'address' field type."